sentry_sdk/integrations/aws_lambda.py

```diff
@@ -37,14 +78,31 @@ def sentry_handler(event, context, *args, **kwargs):
 
         # If an integration is there, a client has to be there.
         client = hub.client  # type: Any
+        configured_time_in_millis = context.get_remaining_time_in_millis()
 
         with hub.push_scope() as scope:
             with capture_internal_exceptions():
                 scope.clear_breadcrumbs()
                 scope.transaction = context.function_name
-                scope.add_event_processor(_make_request_event_processor(event, context))
+                scope.add_event_processor(_make_request_event_processor(event, context, configured_time_in_millis))
 
             try:
+                # Checking if parameter to check timeout is set True
+                if integration.get_check_timeout_error():
+                    # Starting the Timeout thread only if the configured time is greater than Timeout threshold value
+                    if configured_time_in_millis > TIMEOUT_THRESHOLD_MILLIS:
```

Comment:
If we do need this threshold, then let's write a single `if` condition == less indentation levels.

```py
if COND_A and COND_B:
   ...
```